docs/decisions/0006-transmit-updates-only.rst

```diff
@@ -0,0 +1,87 @@
+Transmitting Only Updates IDA
+-----------------------------
+
+Status
+======
+
+Draft
```

Comment:
If you submit this PR it won't be a draft, it will be accepted because it's implemented here :)

Reply:
made it a draft while we go through review 😄